torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2623,6 +2623,7 @@ static void parallelizeOuterLoops(LoopNest& l, Bufs&& bufs) {
 
 StmtPtr TensorExprKernel::transformLoops(BackendType backendType, StmtPtr st) {
   torch::jit::tensorexpr::LoopNest l(st, bufOutputs_);
+  LoopNest::sanitizeNames(l.root_stmt());
```

Comment:
One other alternative is to call `sanitizeNames` from the `LoopNest` constructor. That way we can always ensure all stmts in `LoopNest` adhere to this.

Reply:
Right, we can do that too, I just hoped that we at some point kill LoopNest entirely :D